src/screening/eid/DnaPcrForm.jsx

```diff
@@ -45,59 +43,56 @@ class DnaPcrForm extends React.PureComponent {
           </FormGroup>
         </Row>
 
-        {(typeof this.props.bled !== 'undefined') &&
-        (this.props.bled === CONCEPTS.False.uuid) &&
-        <Row>
-          <FormGroup controlId="formReasonForNoSample">
-            <Col componentClass={ControlLabel} sm={2}>
-              Reason for no sample
-            </Col>
-            <Col sm={10}>
-              <Obs
-                concept={CONCEPTS.ReasonForNoSample.uuid}
-                path="dna-pcrdr-reason-no-sample"
-                conceptAnswers={FORM_ANSWERS.eidNoSampleAnswers}
-              />
-            </Col>
-          </FormGroup>
-        </Row>
-        }
+        <span style={{ display: (typeof this.props.bled !== 'undefined') && (this.props.bled === CONCEPTS.False.uuid) ? 'block' : 'none' }}>
```

Comment:
@mogoodrich, I wanted to extract this to a util function, but it turns out that when the page is loads the first time the function is only called once, and by then the props are still undefined, so the behavior isn't perfect.
We could extract it to utils if we add it to the state, so once the props change we call the function again.
It's fine for now, but if we are going to have more `display: (typeof this.props.bled !== 'undefined') && (this.props.bled === CONCEPTS.False.uuid) ? 'block' : 'none' }` it' would be worth extracting it.

Reply:
Makes sense, thanks @jomadoye !